### DISCUSSION TITLE
No attribute for actual currency symbol (€, $, £, ecc)?

### DISCUSSION THREAD

**Discussion by Saturnix** [django-money/django-money#640] [Q&A]
`my_model.value.currency.code` gives back EUR, USD, ecc...

But what if I want the actual currency symbol (€, $, £, ecc), to be used as a string in Python code? I have inspected all properties of `currency `and the symbol is not there. Is this available as a feature? If not, it's a pretty important feature.

Thanks!

**Comment by benjaoming** [comment]
You could write your own template filter for this. It could look up the currency code in a dictionary and return the currency symbol, defaulting to the currency code.

If you want the symbol to be built into the library, you should look towards pymoneyed: https://github.com/py-moneyed/py-moneyed
